File: calculator-tr.cs
using System;

class Program {
    static void Main() {
        while (true) {
            Console.WriteLine("----------HESAP MAKİNESİ----------");

            Console.Write("İlk rakamı giriniz: ");
            int a = int.Parse(Console.ReadLine());

            Console.Write("İkinci rakamı giriniz: ");
            int b = int.Parse(Console.ReadLine());

            Console.Write("Yapmak istedeğiniz islem türünü seçiniz (+,-,*,/): ");
            string islem = Console.ReadLine();

            if (islem == "+") {
                Console.WriteLine("Sonuc: " + (a + b));
            } else if (islem == "-") {
                Console.WriteLine("Sonuc: " + (a - b));
            } else if (islem == "*") {
                Console.WriteLine("Sonuc: " + (a * b));
            } else if (islem == "/") {
                Console.WriteLine("Sonuc: " + ((float)a / b));
            } else {
                Console.WriteLine("Geçersiz islem");
            }

            Console.WriteLine("İslem tamamlandı.\n");
        }
    }
}

File: calculator-tr.cpp
#include <iostream>
using namespace std;

int main() {
    while (true) {
        int a, b;
        char islem;

        cout << "----------HESAP MAKİNESİ----------\n";
        cout << "İlk rakamı giriniz: ";
        cin >> a;
        cout << "İkinci rakamı giriniz: ";
        cin >> b;
        cout << "Yapmak istedeğiniz islem türünü seçiniz (+,-,*,/): ";
        cin >> islem;

        if (islem == '+') {
            cout << "Sonuc: " << a + b << endl;
        } else if (islem == '-') {
            cout << "Sonuc: " << a - b << endl;
        } else if (islem == '*') {
            cout << "Sonuc: " << a * b << endl;
        } else if (islem == '/') {
            cout << "Sonuc: " << (float)a / b << endl;
        } else {
            cout << "Geçersiz islem\n";
        }

        cout << "İslem tamamlandı.\n\n";
    }
    return 0;
}
